### PR TITLE
Add macro page

### PR DESCRIPTION
**Pull request by NejcZdovc**
PR based on #118. For simplicity Add page is basically only automatic add of the empty macro. After addition you are redirected to the edit page.  There is already opened new macro action waiting for you so you can start adding macros. One thing that is kinda bothering me is that when you add macro action, add action is closed. So if you would like to add 10 actions, you need to click on add new 10 times. Should we leave new action item opened until you click cancel? @mondalaci

Also in this PR I moved list of macros to the new component, so that can be used all around without macro component dependency.

### CODE REVIEW

**Comment by fjozsef** [comment]
Why should we create a page for it? I think it should be just an event handler for the add macro button.

**Comment by NejcZdovc** [comment]
You are completely correct. I was looking this trough component (re-usability) view, but this way we can loose a lot of unnecessary code. Fix is already done.

**Comment by NejcZdovc** [comment]
As discussed with @mondalaci we will not be opening action item, but focus name of the newly added macro. Already implemented.

File: src/components/macro/list/macro-list.component.ts [hunk 1]
```diff
@@ -37,27 +29,38 @@ import { getMacro } from '../../store/reducers/macro';
                 height: '*'
             })),
             transition('inactive <=> active', animate('500ms ease-out'))
+        ]),
+        trigger('togglerNew', [
+            state('void', style({
+                height: '0px'
+            })),
+            state('active',   style({
```
Comment (fjozsef) [inline src/components/macro/list/macro-list.component.ts]: Remove mutliple spaces

Comment (NejcZdovc) [inline src/components/macro/list/macro-list.component.ts]: Done

**Pull request merged**